ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c

```diff
@@ -210,15 +264,16 @@ is_getOutCaps (void **pdata, const GstTensorsConfig * config)
   GstCaps *caps;
   char *str;
 
-  if (idata->mode == MODE_TFLITE_DEEPLAB) {
-    g_return_val_if_fail (config != NULL, NULL);
-    GST_INFO ("Num Tensors = %d", config->info.num_tensors);
-    g_return_val_if_fail (config->info.num_tensors >= 1, NULL);
+  g_return_val_if_fail (config != NULL, NULL);
+  GST_INFO ("Num Tensors = %d", config->info.num_tensors);
+  g_return_val_if_fail (config->info.num_tensors >= 1, NULL);
 
-    if (idata->width == 0 || idata->height == 0) {
-      idata->width = config->info.info[0].dimension[1];
-      idata->height = config->info.info[0].dimension[2];
-    }
+  if (idata->mode == MODE_SNPE_DEPTH) {
```

Comment:
I think this should be `MODE_SNPE_DEEPLAB`

Reply:
Right, thanks.